README.rst

```diff
@@ -1,4 +1,4 @@
-.. image:: doc/_static/dipy-banner.png
+.. image:: https://elef.soic.indiana.edu/dipy/static/images/dipy-logo.png
```

Comment:
Hi Jon. Do not add this link there. This website is temporary. I sent you the png in an email. You can either put the png in a public space like figshare. Or reduce its size (if needed) and add it in doc/_static/dipy-logo.png 
If you go with the second option make sure the png is not larger than 200KBytes.

Reply:
Done in commit ac22727.